core/modules/parser/QSMySqlListener.cc

```diff
@@ -525,13 +545,30 @@ class LogicalOperatorCBH : public BaseCBH {
 };
 
 
+class BitOperatorCBH : public BaseCBH {
+public:
+    enum OperatorType {
+        LEFT_SHIFT,
+        RIGHT_SHIFT,
+        AND,
+        XOR,
+        OR,
+    };
+
+    virtual void handleBitOperator(OperatorType operatorType) = 0;
+};
+
+
 class MathOperatorCBH : public BaseCBH {
 public:
     enum OperatorType {
         SUBTRACT,
         ADD,
-        DIVIDE,
+        DIVIDE, // `/` operator
         MULTIPLY,
+        DIV,    // `DIV` operator
+        MOD,    // `MOD` operator
```

Comment:
Do you really need these trailing comments? I think everything is obvious w/o them.

Reply:
it seemed worth being explicit since there is also e.g. `MODULO`, the `%` operator in that enum.

Reply:
👍